enginetest/queries/view_queries.go

```diff
@@ -20,6 +20,36 @@ import (
 )
 
 var ViewScripts = []ScriptTest{
+	{
+		Name: "existing views",
+		SetUpScript: []string{
+			"create view v as select 1;",
+			"create table t (i int);",
+			"insert into t values (1);",
+		},
+		Assertions: []ScriptTestAssertion{
+			{
+				Query: "create view if not exists v as select 2;",
```

Comment:
Sorry I didn't notice this while looking at the parser updates... it looks like we are slightly off of MySQL's syntax here... it seems like support for `IF NOT EXISTS` was added very recently, in MySQL 9.1. The docs show that that the form should be `CREATE IF NOT EXISTS VIEW v AS SELECT 2;`:
https://dev.mysql.com/doc/refman/9.1/en/create-view.html

I haven't tested this with MySQL 9.x, so it could be that the documentation is incorrect. I know that syntax is a little inconsistent with `CREATE TABLE IF NOT EXISTS`, too.